CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing
+
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a
+Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+
+When you submit a pull request, a CLA bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+## Instructions
+
+Follow these instructions to download and run the sample locally.
+
+1. Install [Node](https://nodejs.org/).
+2. Clone and download this repository.
+3. Navigate to the root of this repository, and install the dependencies: `npm install`
+4. Start the application: `npm start`
+5. Run tests: `npm test`
+6. Run linting: `npm run lint`
```

Comment:
Do we need a single command that does linting / testing and starts the project for anyone who wants to contribute?

Reply:
actually we have no tests or linting script atm. Suppose I should just remove this part?

Reply:
In that case please remove.